unicode-data/unicode-data.cabal

```diff
@@ -33,7 +33,6 @@ extra-source-files:
     Changelog.md
     README.md
     NOTICE
-    ucd.sh
```

Comment:
What caused this change?

Reply:
Because I moved `unicode-data` package to its own subdir. I also plan to move ucd2haskell to its proper package. What do you think?

Reply:
I do not have a strong opinion. A few things that come to mind:

* Too many packages?
* The new package may not be a general purpose package, it would be used only by `unicode-data*` packages.

Reply:
I do not plan to publish it, but keep it an internal package.

Reply:
Should be fine, I guess. We have benchmarks and tests as separate internal packages in streamly, though we ship the code in the same distribution.

Reply:
Let’s keep it like this atm. See #74